src/styles/blocks/icons.scss

```
@@ -0,0 +1,51 @@
+.icon {
+  width: 24px;
+  height: 24px;
+
+  &:last-child {
+    margin-left: 24px;
+  }
+
+  &--number {
+    display: none;
+    opacity: 0;
+    transition: opacity 0.4s ease;
+    margin-top: 5px;
+    margin-left: -160px;
+    color: $text-color-light;
+    width: 200px;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 2px;
+  }
+
+  &--phone {
+    transition: opacity 0.4s;
+    background-repeat: no-repeat;
+    background-image: url(../img/icons/icon-phone.svg);
+    background-position: center;
+
+    &:hover .icon--number {
+      display: flex;
+      opacity: 1;
+    }
```

You should nest this:

```suggestion
    &:hover {
      &.icon--number {
        display: flex;
        opacity: 1;
      }
   }
```

---

Не работает, сработает только если внутри ховера оставить .icon--number {
//properties
}